### DISCUSSION TITLE
全機模組化的 Fairphone 4 實機上手：可自行維修拆解，對消費者、環境最公平的環保手機

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#1006] [科技新知]
他們家的產品一向都以環保為賣點，從使用材料到製造過程的碳排都很講究，新推出的Fairphone 4的零件可更替性更高了，幾乎整支手機的零件都可由使用者自行更換，也讓其淘汰年限拉的更長，但可惜的是他的價錢和同級手機比起來實在太貴了，螢幕更停留在上一代的60Hz水滴屏，難道環保產品都一定要那麼坑的嗎？
[文章連結](https://tinyurl.com/2z84uwus)

**Comment by peggy0329** [comment]
其實這就是一體兩面的事情：環保產品製作成本高、售價貴;不環保產品製造過程費用較低、售價較低
我覺得環保產品都很貴的主要原因也包括製作過程中時勞力成本高、高品質以及顧慮環境的製作過程
不環保的產品主要也都是靠壓低成本才可以販售較低的價格
但壓低成本可能就是會用低薪雇用員工或工廠製作過程沒有一些環保檢測或過濾的過程
